### ISSUE TITLE
Train from scratch.

### ISSUE THREAD

**Issue by herrickli** [amdegroot/ssd.pytorch#407]
If I want to train pascal_voc dataset from scratch, what tricks should be used and what are the lowest loss of conf and loc. 
I use `lr=1e-4`, `batch_size = 16`, after 20 epoches, the `conf_loss` is about `5`, the `loc_loss` is about `1.7`, and they don't seem to be decreasing any more, and I evalute the model, it has a bad preformance, nearly all the biggest confidence of priors are `BACKGROUND`.

![issue](https://user-images.githubusercontent.com/43718467/65020826-82705b00-d960-11e9-888f-41c737e5b439.png)
So what should I do to make it better?
Could someone help me please? 
Many thanks!
